src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -462,7 +466,7 @@ public static function get_user_data() {
 		}
 
 		$config   = array();
-		$user_cpt = static::get_user_data_from_wp_global_styles( wp_get_theme() );
+		$user_cpt = static::get_user_data_from_wp_global_styles( get_stylesheet() );
```

Comment:
Nit-pick, but can you update this in the tests calling this method as well? Just so that we then consistently use the method with a stylesheet in core, as that should be now encouraged.